### DISCUSSION TITLE
Ubuntu 24.03  snap package - :w causes file sort

### DISCUSSION THREAD

**Discussion by whodidwhat** [helix-editor/helix#11017] [General]
### Summary

the helix snap sorts the file whenever the :w command is entered

### Reproduction Steps

 

I tried this:

open file
issue :w

I expected this to happen:
file to save

Instead, this happened:
file sorts then saves

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux

### Terminal Emulator

gnome

### Installation Method

snap

### Helix Version

24.03

**Comment by krish-r** [comment]
By any chance, is the file named `todo.txt` or something similar?

**Comment by dead10ck** [comment]
I'm guessing this is an auto formatter going haywire.

**Comment by the-mikedavis** [comment]
I believe this is the same as https://github.com/helix-editor/helix/issues/10113. The fix for that isn't included in a release yet
